templates/strings-structured.stencil

```diff
@@ -0,0 +1,233 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+{% if structuredStrings %}
```

Comment:
It seems like you fixed an issue when previously using `if structuredStrings.strings` in previous commit (5aec504abf8aba42b08129750e68a087759ded03)… but the travis build before this fixing commit was still successful when you had that.

So maybe, if with the old code you had an issue and not the expected generated output, but the tests did still pass at that time despite the issue… that means that we are missing a Unit Test to cover that use case and prevent that this issue reappear in the future?